cypress/e2e/wip/dtrader/contractsDigitsEvenOddresponsive.cy.js

```diff
@@ -0,0 +1,42 @@
+import '@testing-library/cypress/add-commands'
+
+const stakeAmount = '5'
+
+describe('QATEST-6332 -  Verify contract for Digits', () => {
+  const select = ['even', 'odd']
```

Comment:
Variable names should be clearer.
```suggestion
  const tradeType = ['even', 'odd']
```